contivctl/commands.go

```diff
@@ -0,0 +1,191 @@
+package contivctl
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+var tenantFlag = cli.StringFlag{
+	Name:  "tenant, t",
+	Value: "default",
+	Usage: "Name of the tenant to create the policy on.",
+}
+
+var Commands = []cli.Command{
+	{
+		Name:  "group",
+		Usage: "Endpoint Group manipulation tools",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "Create an endpoint group",
+				ArgsUsage: "[group] [network]",
+				Flags: []cli.Flag{
+					tenantFlag,
+					cli.StringFlag{
+						Name:  "policy, p",
+						Usage: "Policy List (separated by commas)",
+					},
+				},
+				Action: CreateEndpointGroup,
+			},
+			{
+				Name:      "delete",
+				Usage:     "Delete an endpoint group",
+				ArgsUsage: "[group]",
+				Flags:     []cli.Flag{tenantFlag},
+				Action:    DeleteEndpointGroup,
+			},
+			{
+				Name:      "list",
+				Usage:     "List endpoint groups",
+				ArgsUsage: " ",
+				Flags: append([]cli.Flag{tenantFlag},
+					cli.BoolFlag{
+						Name:  "all, a",
```

Comment:
If we allow 'all' - does it mean we can specify a specific element if '-a' flag is not specified? Or is that a default flag?